app/Snapshot.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Snapshot extends Model
+{
+    protected $guarded = [];
+
+    public function scopeToday($query)
+    {
+        return $query->whereDate('created_at', '=', now()->format('Y-m-d'));
```

Comment:
Should be `snapshot_date` instead.

```suggestion
        return $query->whereDate('snapshot_date', '=', now()->format('Y-m-d'));
```